src/user/Skill.ts

```diff
@@ -0,0 +1,35 @@
+export class Skill {
```

Comment:
Might wanna add some get methods to retrieve the value of these private members.